src/main/java/com/smartbear/readyapi/client/execution/CodegenBasedTestServerApi.java

```
@@ -282,4 +290,79 @@ private GenericType getReturnTypeHarLogRoot() {
     public void setBasePath(String basePath) {
         apiClient.setBasePath(basePath);
     }
+
+    @Override
+    public ProjectResultReport postProject(File file, boolean async, HttpBasicAuth auth) throws ApiException {
+
+        setAuthentication(auth);
+
+        List<Pair> queryParams = new ArrayList<>();
+        queryParams.add(new Pair("async", String.valueOf(async)));
+
+        String path = ServerDefaults.SERVICE_BASE_PATH + "/executions";
+        String type = "application/xml";
+
+        try {
+            // composite project?
+            if (file.isDirectory()) {
```

Should we throw an exception with user friendly message if file doesn't exist ?